.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,17 @@
+name: auto-merge
```

Comment:
Should this be called `auto-approve` instead of `auto-merge`? (If you agree, `auto-merge` is mentioned in the file name and twice in the names in this file, excluding the label.)
It may also be helpful to add a comment saying that auto-approve in combination with Renovate settings will lead to automerging of the PR.